lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU

```diff
@@ -90,7 +90,7 @@ VAR
     fbGige: FB_PPM_Gige;
 
     {attribute 'pytmc' :='pv: FWM'}
-    fbFlowMeter: FB_AnalogInput := (iTermBits:=12, fTermMax:=60, fTermMin:=0, fResolution:=0.1);
+    fbFlowMeter: FB_AnalogInput := (iTermBits:=15, fTermMax:=60, fTermMin:=0);
```

Comment:
good catch :+1:

Reply:
I think this is a result of your bit catch in other sensors with the 3052. 😉

Reply:
You've caught me the same way I caught you, so we're even